pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
     "prometheus-client==0.21.0",
     "ruamel-yaml>=0.18.10",
     "redis>=6.0.0",
+    "huggingface-hub>=0.30.2",
```

Comment:
I had to install this in order to login.  With the gemma model default, it required a login to check that I had accepted model terms